metaflow/plugins/cards/card_modules/components.py

```diff
@@ -103,15 +112,15 @@ def __init__(self, data=[[]], headers=[]):
             self._data = data
 
     @classmethod
-    def from_dataframe(cls, dataframe=None, truncate=True):
+    def from_dataframe(cls, dataframe=None, truncate: bool = True):
```

Comment:
Was `dataframe: pd.DataFrame` not used here because it would require a new import statement `import pandas as pd`?

Reply:
Yes, I would like to avoid the import. Maybe I can use a string here too.